Tuesdays/Week-Class/Inheratance/Assets/Scripts/PlayableCharacter.cs

```diff
@@ -6,21 +6,24 @@ public abstract class PlayableCharacter : MonoBehaviour, IDamagable{
 	[SerializeField] int speed;
 	[SerializeField] int maxHP;
 	int currHP;
+	public delegate void HPDelegate(int hp);
+	public event HPDelegate HPChangeHandler; 
```

Comment:
event should always be names at something that already happened in the past:
"PlayerJumped" / "HpChanged" /"HpWasReduces" / "HpIncreased" etc.
get as specific as you can